test/e2e/performanceprofile/functests/10_performance_ppc/10_ppc_suite_test.go

```diff
@@ -0,0 +1,58 @@
+package __performance_ppc
```

Comment:
we should probably rename all the other suites to read like `00_config` and so forth, and this should probably be done in its own prerequisite PR